apps/src/storage/firebaseUtils.js

```diff
@@ -94,6 +95,28 @@ export function getProjectDatabase() {
   return getFirebase().child(path);
 }
 
+function getSharedFirebase() {
+  let sharedFb = sharedFirebaseCache;
+  if (!sharedFb) {
+    if (!config.firebaseSharedAuthToken) {
+      throw new Error(
+        'Error connecting to Firebase: Shared token not specified'
```

Comment:
please name the CDO variable here so that developers without it know how to fix it.